Framework/DataObjects/src/PeaksWorkspace.cpp

```diff
@@ -149,6 +150,28 @@ void PeaksWorkspace::removePeak(const int peakNum) {
   peaks.erase(peaks.begin() + peakNum);
 }
 
+/** Removes multiple peaks
+* @param badPeaks peaks to be removed
+*/
+void PeaksWorkspace::removePeaks(const std::vector<int> &badPeaks) {
+  if (badPeaks.size() == 0)
+    return;
+  int i = 0;
```

Comment:
Can we move `i` into the for loop as it is only used there
`for (int i = 0; first < end....`